backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/IInvoiceCandBL.java

```diff
@@ -69,6 +72,10 @@ public interface IInvoiceCandBL extends ISingletonService
 
 	boolean isAllowedToCreateInvoiceCandidateFor(Object model);
 
+	Timestamp getBaseLineDate(@NonNull PaymentTerm paymentTerm, @NonNull I_C_Invoice_Candidate ic);
```

Comment:
I would prefer to use Instant instead of Timestamp where applicable...

Reply:
also feel free to leave it like it is now, it the change is not quick

Reply:
Too complicated now..